src/fsharp/est.fs

```diff
@@ -31,6 +31,8 @@ module internal ExtensionTyping =
         /// The absolute path name to where approvals are stored
         let ApprovalsAbsoluteFileName = Path.Combine(System.Environment.GetFolderPath(System.Environment.SpecialFolder.LocalApplicationData), @"Microsoft\VisualStudio\12.0\type-providers.txt")
```

Comment:
Curious: why is 12.0 hard coded here vs. the current version of VS or a non-VS location?